pkg/userdata/kubeadm.go

```diff
@@ -23,10 +24,10 @@ type Kubeadm struct {
 	Configuration    runtime.Object `yaml:"-"`
 	ConfigurationStr string         `yaml:"configuration"`
 
-	ExtraArgs             []string `yaml:"extraArgs,omitempty"`
-	CertificateKey        string   `yaml:"certificateKey,omitempty"`
-	IgnorePreflightErrors []string `yaml:"ignorePreflightErrors,omitempty"`
-	bootstrap             bool
+	ExtraArgs             []string     `yaml:"extraArgs,omitempty"`
+	CertificateKey        string       `yaml:"certificateKey,omitempty"`
+	IgnorePreflightErrors []string     `yaml:"ignorePreflightErrors,omitempty"`
+	Token                 *token.Token `yaml:"initToken,omitempty"`
```

Comment:
Thus far we have called this a "bootstrap token". I have not particular favorite between a "bootstrap token" and "init token". Either way we should align what we are calling this externally with what we are calling it internally. We have `BootstrapTTL` and `IsBootstrap` so maybe we call it `bootstrapToken`?

Reply:
I was using bootstrapToken, but later changed it to initToken because there was already a notion of a bootstrap token within the kubeadm config. The change to `init` seemed to set it apart enough so it wouldn't be confused with the kubeadm bootstrap token. I can start going further down to replace bootstrap with init if that'd help for consistency?

Reply:
We can think about it during refactor